galsim/inclinedsersic.py

```diff
@@ -0,0 +1,216 @@
+# Copyright (c) 2012-2016 by the GalSim developers team on GitHub
+# https://github.com/GalSim-developers
+#
+# This file is part of GalSim: The modular galaxy image simulation toolkit.
+# https://github.com/GalSim-developers/GalSim
+#
+# GalSim is free software: redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions, and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+#
+"""@file inclinedsersic.py
+
+InclinedSersic is a class representing a (possibly-truncated) Sersic profile inclined to the LOS.
+"""
+
+from galsim import GSObject
+import galsim
+
+from . import _galsim
+
+
+class InclinedSersic(GSObject):
+    """A class describing an inclined sersic profile. This class is general, and so for certain
+    special cases, more specialized classes will be more efficient. For the case where n==1
+    with no truncation, the InclinedExponential class will be much more efficient. For the case
+    where the inclination angle is zero (face-on), the Sersic class will be slightly more efficient.
+
+    The Inclined Sersic surface brightness profile is characterized by four properties: its
+    Sersic index `n', its inclination angle (where 0 degrees = face-on and 90 degrees = edge-on),
+    its scale radius, and its scale height. The 3D light distribution function is:
+
+        I(R,z) = I_0 / (2h_s) * sech^2 (z/h_s) * exp[-b*(r/r_s)^{1/n}]
```

Comment:
I think it isn't much an issue for the requested use case as Lance always includes an additional bulge component (a standard Sersic profile) which is cuspy near the centre; this profile isn't intended to represent the whole galaxy, but only the disk component.